README.md

```diff
@@ -430,17 +428,9 @@ await octokit.request("POST /repos/{owner}/{repo}/issues", {
 });
 ```
 
-The request is authenticated, retried, and throttled transparently by the `octokit` instance. The `accept` and `user-agent` headers are set as well.
-
 The 1st argument is the REST API route as listed in GitHub's API documentation. The 2nd argument is an object with all parameters, independent of whether they are used in the path, query, or body.
 
-See the standalone [`@octokit/request`](https://github.com/octokit/request.js#readme) module for more details on the `.request()` APIs.
-
-If you only need the request options, but want to send the actual request using your own method, use `octokit.request.endpoint(route, parameters)`. See the standalone [`@octokit/endpoint`](https://github.com/octokit/endpoint.js#readme) module for more details on the `.endpoint()` APIs.
-
-#### Pagination
-
-**plugin**: [`@octokit/plugin-paginate-rest`](https://github.com/octokit/plugin-paginate-rest.js/#readme).
+#### Pagination ([`@octokit/plugin-paginate-rest`](https://github.com/octokit/plugin-paginate-rest.js/#readme))
 
 All `octokit.rest.*.list*` endpoints return the first 30 items by default. If you want to retrieve all items, you an use the pagination API.
```

Comment:
Does pagination only apply to `rest`? If so, that's a pretty big differentiator that should be called out explicitly here and above to support users deciding which to use.  If not, suggest having two examples, one for `rest` and one for `request` or pointing out that "the `request` form is exactly the same just pass the `request` object to `iterator`)

Reply:
Yes good idea, let's add two examples.

How about we make it clear in this paragraph that pagination works for both, `the `octokit.rest.*.list*` methods as well as `octokit.request()` for endpoints that paginate, maybe like this?

```suggestion
All REST API endpoints that paginate return the first 30 items by default. If you want to retrieve all items, you an use the pagination API. The pagination API expects the REST API route as first argument, but you can also pass any of the `octokit.rest.*.list*` methods for convenience and better code readability.
```

Then we could add an additional example to the end?

````md
Instead of passing `octokit.rest.issues.listForRepo`, you can pass `"GET /repos/{owner}/{repo}/issues"`, both behave the same way. For example

```js
const issues = await octokit.paginate("GET /repos/{owner}/{repo}/issues", {
  owner: "octocat",
  repo: "hello-world",
  per_page: 100,
});
```
````